app/components/Pending.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import React, { Component } from "react";
+import { StyleSheet, Text } from "react-native";
+import { H2, View } from "native-base";
+
+import { Background, Dot } from "components";
+import { colours, standards } from "styles";
+
+type Props = {
+  condition: string,
+};
+class Pending extends Component<Props> {
+  render() {
+    const { condition } = this.props;
+
+    return (
+      <Background noRalph={true} style={styles.background}>
+        {this._renderDots(condition)}
+        {this._renderConditionText(condition)}
+        <View style={styles.dotsView} />
+      </Background>
+    );
+  }
+
+  _renderDots(condition: string) {
+    if (condition !== "Loading") {
```

Comment:
I feel like this might read slightly more clearly with a positive condition first:

```js
if (condition === "Loading") {
  return dots;
}

return somethingElse;
```

What do you think?